dnf.spec

```diff
@@ -85,6 +85,8 @@ Requires:       python3-%{name} = %{version}-%{release}
 %if 0%{?rhel} && 0%{?rhel} <= 7
 Requires:       python-dbus
 Requires:       %{_bindir}/sqlite3
+%elif 0%{?fedora}
+Recommends:     (%{_bindir}/sqlite3 if (bash-completion && python3-dnf-plugins-core))
```

Comment:
I am really sorry but operator `&&` is not supported. See https://rpm-software-management.github.io/rpm/manual/boolean_dependencies.html. May I ask you to replace `&&` by `and`?